app/plugin_api.go

```diff
@@ -81,12 +99,24 @@ func (api *PluginAPI) GetSession(sessionId string) (*model.Session, *model.AppEr
 }
 
 func (api *PluginAPI) GetConfig() *model.Config {
-	return api.app.GetSanitizedConfig()
+	cfg := api.app.GetSanitizedConfig()
+
+	if pluginConfig, isOk := cfg.PluginSettings.Plugins[api.manifest.Id]; isOk {
```

Comment:
Pursuant to @hanzei's point above, if `isOk` is false here, we'll also never set the defaults.

Reply:
@lieut-data yes, I will update GoDoc and make changes according to the @hanzei comments. Thank you guys.